opts/opts.go

```diff
@@ -24,6 +24,10 @@ func ListVar(values *[]string, names []string, usage string) {
 	flag.Var(newListOptsRef(values, nil), names, usage)
 }
 
+func MapVar(values *map[string]string, names []string, usage string) {
```

Comment:
@LK4D4 You're right, theres no need for one. I was trying to be consistent with the rest of the types. Changed it just the map type now